pkg/sfu/streamallocator.go

```diff
@@ -808,6 +884,9 @@ func (s *StreamAllocator) allocate() {
 	// So, track total requested bandwidth and mark DEFICIENT state if the total is
 	// above the estimated channel capacity even if the optimal signal is true.
 	//
+	var pausedTracks map[string][]string
```

Comment:
since the protocol and downstream do not require map[publisher][]trackid format, does it make sense to simplify it here as well?

Reply:
Left a note about using proto specific type deep in code. I can change it if that is okay.